lib/matplotlib/animation.py

```diff
@@ -640,45 +629,6 @@ def _args(self):
                 '-vframes', str(self._frame_counter)] + self.output_args
 
 
-# Base class of avconv information.  AVConv has identical arguments to FFMpeg.
-@cbook.deprecated('3.3')
-class AVConvBase(FFMpegBase):
```

Comment:
Trying to sneak that deprecation deletion in here ;-)

Reply:
Wow, what a mistake. I removed the 2 commits from my other branch.